### PR TITLE
[Osquery] Adjustments to saved queries

### PR DESCRIPTION
**Pull request by tomsonpl**
- change saved queries json files

### CODE REVIEW

File: packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "attributes": {
+        "created_at": "2022-05-19T10:33:59.162Z",
+        "created_by": "elastic",
+        "description": "Get RAM details for MacOS systems.",
```
Comment (melissaburpo) [inline packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json:5]: Update `MacOS` here to `Linux`

File: packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "ecs_mapping": [],
+        "id": "system_memory_mac_elastic",
```
Comment (melissaburpo) [inline packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json:7]: rename to `system_memory_linux_elastic`

**Review by melissaburpo — COMMENTED** [review]
Added comments for two small updates to make. Otherwise, this looks good!

File: packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json (hunk 1 above)
Comment (tomsonpl) [reply packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json]: 👍  thanks Melissa :)

File: packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json (hunk 2 above)
Comment (tomsonpl) [reply packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-315bfda0-d75f-11ec-879b-83915b27217e.json]: 👍  thanks Melissa :)

**Pull request merged**